components/FbxTabs/FbxTabs.vue

```
@@ -0,0 +1,65 @@
+<template>
+  <div class="fbx-tabs">
+    <div
+      class="fbx-tab"
+      v-for="(tab, i) in tabs"
+      :key="tab"
+      :class="{ 'active': currentTabIndex === i }"
+      @click="onTabClick(i)"
+    >
+      {{ tab }}
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "FbxTabs",
+  data() {
+    return {
+      currentTabIndex: 0,
+    }
+  },
+  props: {
+    tabs: Array,
```

Should add `required` safety so it will fail loudly.